packages/sdk/src/conversation/interface.ts

```diff
@@ -125,18 +145,18 @@ export interface TeamsFxBotCommandHandler {
   /**
    * The command name or RegExp pattern that can trigger this handler.
```

Comment:
"command name" can be removed in comment.

Reply:
Good catch, fixed.